RELEASE.md

```diff
@@ -61,7 +76,11 @@ To create a new version (with tag), follow all these steps! They are necessary t
 
 * Update `Version` in `config/vars.go` - change it to the next version (eg. from `v2.3.1-dev` to `v2.3.1`), and commit
 * Create a [signed git tag](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-tags): `git tag -s v2.3.1`
-* Now push to main and push the tag: `git push && git push --tags`
-* Update `Version` in `config/vars.go` to next patch with `dev` suffix (eg. `v2.3.2-dev`), and commit
+* Update the `stable` branch:
+  * `git checkout stable`
+  * `git merge tags/v2.3.1 --ff-only`
+  * `git checkout main`
+* Update `Version` in `config/vars.go` to next patch with `dev` suffix (eg. `v2.3.2-dev`), commit to main and push to Github
+* Now push the main and stable branch, as well as the tag to Github: `git push origin main stable --tags`
 
 Now check the Github CI actions for release activity: https://github.com/flashbots/mev-boost/actions - CI builds and pushes the Docker image, and prepares a new draft release in https://github.com/flashbots/mev-boost/releases. Open it, generate the description, review, add signoffs and testing, and publish.
```

Comment:
Now check the Github CI actions for release activity: https://github.com/flashbots/mev-boost/actions

- CI builds and pushes the Docker image, and prepares a new draft release in https://github.com/flashbots/mev-boost/releases. 
- Open it, generate the description, review, add sign-offs, tests, and publish.